File: src/main/java/com/antonio32a/examplemod/commands/ExampleCommand.java
package com.antonio32a.examplemod.commands;

import club.sk1er.mods.core.ModCore;
import com.antonio32a.examplemod.ExampleMod;
import net.minecraft.command.CommandBase;
import net.minecraft.command.ICommandSender;

public class ExampleCommand extends CommandBase {
    @Override
    public String getCommandName() {
        return "example";
    }

    @Override
    public String getCommandUsage(ICommandSender sender) {
        return "/example";
    }

    @Override
    public int getRequiredPermissionLevel() {
        return -1;
    }

    @Override
    public void processCommand(ICommandSender sender, String[] args) {
        ModCore.getInstance().getGuiHandler().open(ExampleMod.getInstance().getConfig().gui());
    }
}

File: src/main/kotlin/com/antonio32a/examplemod/commands/ExampleCommand.kt
package com.antonio32a.examplemod.commands

import com.antonio32a.examplemod.ExampleMod
import gg.essential.api.EssentialAPI
import gg.essential.api.commands.Command
import gg.essential.api.commands.DefaultHandler

class ExampleCommand : Command("example") {
    @DefaultHandler
    fun handle() {
        EssentialAPI.getGuiUtil().openScreen(ExampleMod.config.gui())
    }
}